client/components/__tests__/Voting.test.js

```diff
@@ -18,28 +18,33 @@ const fakeStore = {
 }
 
 describe('<Voting />', () => {
+  postVotes.mockReturnValue(Promise.resolve())
+  postVotesTie.mockReturnValue(Promise.resolve())
+
   it('renders on the screen', async () => {
     render(
       <Provider store={fakeStore}>
         <Voting cat={objTwoPet.cat} dog={objTwoPet.dog} />
       </Provider>
     )
-    postVotes.mockReturnValue(Promise.resolve())
-    const button = screen.getByRole('button', { name: 'Pick Me!' })
-    await userEvent.click(button)
+    await userEvent.click(screen.getByRole('button', { name: 'Pick Me!' }))
+
     expect(postVotes).toHaveBeenCalledWith(objTwoPet.cat.id)
   })
+
   it('after button click expect new pets to render', async () => {
     render(
       <Provider store={fakeStore}>
         <Voting cat={objTwoPet.cat} dog={objTwoPet.dog} />
       </Provider>
     )
-    const button = screen.getByRole('button', { name: /skip/i })
-    expect(button).toBeInTheDocument()
 
-    await userEvent.click(button)
+    await userEvent.click(screen.getByRole('button', { name: /skip/i }))
 
+    expect(postVotesTie).toHaveBeenCalledWith(
+      objTwoPet.cat.id,
+      objTwoPet.dog.id
+    )
     expect(fakeStore.dispatch).toHaveBeenCalled()
   })
 })
```

Comment:
Nice tests, you need another one for when 'No, click me' is clicked as that is currently not being tested